tests/lifecycle/tests/lifecycle_pod_high_availability.go

```diff
@@ -47,12 +47,12 @@ var _ = Describe("lifecycle lifecycle-pod-high-availability", func() {
 		By("Start lifecycle lifecycle-high-availability test")
 		err = globalhelper.LaunchTests(
 			[]string{lifeparameters.LifecycleTestSuiteName},
-			lifeparameters.SkipAllButPodHighAvailabilityRegex)
+			globalhelper.GetStringOfSkipTcs(lifeparameters.SkipsTcsSlice, lifeparameters.PodHighAvailabilityName))
```

Comment:
might be cleaner to call this once and assign it to a variable since it's called multiple times with the same params

Reply:
good comment, will upload a better version soon, thanks!